.bazelrc

```diff
@@ -1,7 +1,7 @@
 # Bazel distributed cache, can be temporarily disabled by passing the following
 # flag: --noremote_accept_cached
-build:darwin --remote_http_cache=https://bazel-cache.da-ext.net
-build:linux --remote_http_cache=https://bazel-cache.da-ext.net/ubuntu_20_04
+build:darwin --remote_cache=https://bazel-cache.da-ext.net/bump
```

Comment:
since this built on windows I'm confident it should also build on macos and linux, so I applied this change. for the windows cache, I removed my `/bump` and instead increased `-v{13=>14}` (the [big comment in ci/configure-bazel.sh](https://github.com/digital-asset/daml/blob/62c9864514d736e1987015877735658ff95a4660/ci/configure-bazel.sh#L57-L71) made me worry that a cleaner (but longer) suffix might break things on windows)